docs/tutorials/start-a-private-network/3keygen.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Generate Your Own Keys
+---
+
+Now that we know the fundamentals and the command line options, it's time to generate our own keys rather than using the well-known Alice and Bob keys. Each person who wants to participate in the blockchain should generate their own keys. This page explains several options for generating keys, and each participant only needs to choose one. Regardless of which option you choose, be sure to record all of the output from this section as you will need it later.
+
+## Option 1: Subkey
+Subkey is a tool the generates keys specifically designed to be used with Substrate. This is the most secure way to generate keys, and is how you should generate keys if you plan to use them in production.
+
+Begin by compiling and install the utility.
+```bash
+cargo install --force subkey --git https://github.com/paritytech/substrate
+```
+
+Generate a mnemonic and see the `sr25519` key and address associated with it. This key will be used by Aura for block production.
+```
+$ subkey generate
+Secret phrase `keep matrix knee meat awake frown rubber position federal easily strategy inhale` is account:
+  Secret seed: 0xb5d5cda89e139aecb67181e11d6d2d90a0cc80106afa035ab19264af7b5e5c0b
+  Public key (hex): 0x8ed5f822065e5824d3e37d9ea36a81eacb98ff1a6fa04bb87d2fa4915e9ed147
+  Address (SS58): 5FHzDem7A5aAq79tuEN9xJuNPXiYfmRQamhumTuqu6i57BuU
+```
+
+Now see the `ed25519` key and address associated with the same mnemonic. This key will be used by GRANDPA for block finalization.
+```
+$ subkey --ed25519 inspect "keep matrix knee meat awake frown rubber position federal easily strategy inhale"
+Secret phrase `keep matrix knee meat awake frown rubber position federal easily strategy inhale` is account:
+  Secret seed: 0xb5d5cda89e139aecb67181e11d6d2d90a0cc80106afa035ab19264af7b5e5c0b
+  Public key (hex): 0xfe68fdff17960cb8d45d861396a64d4086997353849403ee3352996ec68ff4af
+  Address (SS58): 5HpHD5YseSWbHfni43Zm2SjtpyqSVmuaNhKkVmivp8L93Trs
+```
+
+## Option 2: Polkadot JS Apps
```

Comment:
```suggestion
## Option 2: Polkadot-JS Apps
```

Reply:
How about put Polkadot JS Apps as Option 1? Since it's more efficient for testing purpose.

Reply:
At the end it doesn't matter for this tutorial. You need to enter the `seed` for becoming a validator. It doesn't matter how you get it. But I think that because this is not a beginners tutorial, it should be okay to introduce a new tool.